core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4565,6 +4575,13 @@ int RootClingMain(int argc,
       CreateDictHeader(*dictStream, main_dictname);
       if (gOptSplit)
          CreateDictHeader(*splitDictStream, main_dictname);
+
+      if (!gOptNoUsingGlobalStd) {
+	AddNamespaceSTDdeclaration(*dictStream);
+	if (gOptSplit) {
+	  AddNamespaceSTDdeclaration(*splitDictStream);
+	}
+      }
```

Comment:
Looks like this entire new blocks has tabs. Could you configure your editor to insert spaces instead?